tests/main/snap-download/task.yaml

```diff
@@ -25,6 +25,12 @@ execute: |
     su -l -c "SNAPPY_USE_STAGING_STORE=$SNAPPY_USE_STAGING_STORE snap download test-snapd-tools" test
     ls /home/test/test-snapd-tools_*.snap
     verify_asserts /home/test/test-snapd-tools_*.assert
+
+    echo "Snap download can download from a different store"
```

Comment:
Thanks! Had the same pending locally but didn't know that there is a demo store we can use for testing.